### DISCUSSION TITLE
Ability to pay for bandwidth

### DISCUSSION THREAD

**Discussion by SoniEx2** [modrinth/.github#7] [Feature requests]
### Is your feature request related to a problem? Please describe.

We don't like ads.

### Describe the solution you'd like

There should be some way to pay for the bandwidth a project uses. This would be an alternative to showing ads on the project's page.

### Describe alternatives you've considered

N/A

### Additional context

N/A

**Comment by Andre601** [comment]
I may be blind, but where exactly are ads displayed?
I have my ad-blocker - which you could use btw if you really hate ads - disabled on the modrinth website, but can't see any ads whatsoever...

**Comment by triphora** [comment]
@Andre601 
![image](https://user-images.githubusercontent.com/60205699/177365304-8789cae5-aee2-4a5d-ac7c-cdf0efe9f0f5.png)
![image](https://user-images.githubusercontent.com/60205699/177365351-10c615c1-ef18-41ad-8783-a89e840d9bcd.png)

**Comment by Andre601** [comment]
🤷 
![image](https://user-images.githubusercontent.com/11576465/177365528-ad536c5e-1dea-4b24-8864-2d6d1c905b98.png)

Maybe a browser thing, idk.
Also, personally, those kinds of ads are - at least for me - the least annoying ones. Small text saying something. There are way worse (and more intrusive) ads IMO that people should worry about.

**Comment by DioEgizio** [comment]
They're so non-intrusive, I don't see them as issues imo

**Comment by Southpaw1496** [comment]
A Modrinth Pro (Modrinth Green?) subscription to get rid of ads and give creators a payout based on some metric, similar to YouTube Red, would be an interesting concept. However, it would take a lot of work, and there are other issues with the site which I think are far more pressing, especially considering how unintrusive the ads are, and the fact that if you really hate them you can turn on an ad blocker.

**Comment by SoniEx2** [comment]
No, this is for creators to get rid of ads on their project pages.

**Reply by gigglingbed** [reply]
like creators pay to not make revenue?

**Reply by SoniEx2** [reply]
no it's called "paying for hosting", optionally based on usage quotas.

**Reply by kyh148** [reply]
IIRC all revenue from creators' pages goes to the creators themselves in the form of payouts, and only that of ads shown in search etc. goes to Modrinth.

**Reply by gigglingbed** [reply]
yeah that would make the most sense 

the idea of ads (and therefore revenue) being optional on mod pages (for the creator to determine) sounds much more useful to me

i'm sure there are plenty of people who have no interest in taking revenue from their mods so the option to either have no ads or to just show ads but not build revenue could be cool?

**Comment by Southpaw1496** [comment]
Given that almost all (if not all) the ad revenue will go to creators, it may be an idea to allow creators to not show ads on their project pages and not receive any payouts, I think YouTube has a similar system.
